perfrepo-cli

```diff
@@ -157,18 +157,18 @@ class TestCLI(GenericCLI):
             try:
                 test_id = int(argv[1])
                 test = self._perf_api.test_get_by_id(test_id)
-                if test is None:
-                    test = self._perf_api.test_get_by_uid(test_id)
-                if test is None:
-                    print("Test not found.")
-                else:
-                    print(test)
             except IndexError:
                 print("No ID specified!", file=sys.stderr)
                 return -1
             except ValueError:
-                print("ID needs to be an integer!", file=sys.stderr)
-                return -1
+                test = self._perf_api.test_get_by_uid(argv[1])
+
+            if test is None:
+                print("Test not found.")
+                return 0
```

Comment:
OK, my proposal here is to have two different error return codes. rc=-1 if data could not be retrieved from perfrepo (not an user input error), rc=-2 for anything that is user input error that will print the help.